src/main/org/nlogo/headless/ChecksumsAndPreviews.scala

```diff
@@ -41,6 +41,8 @@ object ChecksumsAndPreviews {
   }
 
   object Previews {
+    def needsManualPreview(previewCommands: String) =
+      previewCommands contains "need-to-manually-make-preview-for-this-model"
```

Comment:
```
scala> def needsManualPreview(previewCommands: String) =
     | previewCommands contains "need-to-manually-make-preview-for-this-model"
needsManualPreview: (previewCommands: String)Boolean

scala> needsManualPreview("need-to-manually-make-preview-for-this-model")
res1: Boolean = true
```
I think it is `java.lang.String.contains` being called here, not `scala.collection.SeqLike.contains`. `contains` reads clearer to me than `containsSlice`, and is probably a bit faster. Would there be any reason to be using `containsSlice` anyway? (A bit clearer, perhaps?)